### PR TITLE
Fix problem with string format

### PR DESCRIPTION
**Pull request by vbauer**
MessageFormat supplied where printf style format expected.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vbauer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/main/java/microsoft/exchange/webservices/data/ExchangeServiceBase.java [hunk 1]
```diff
@@ -312,7 +316,7 @@ protected void internalProcessHttpErrorResponse(HttpWebRequest httpWebResponse,
       }
 
       this.traceMessage(responseTraceFlag,
-          String.format("Account is locked. Unlock URL is {0}", accountUnlockUrl));
+          String.format("Account is locked. Unlock URL is %s", accountUnlockUrl));
```
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/ExchangeServiceBase.java]: why not use string concatenation here too?

**Comment by vbauer** [comment]
I've just updated PR, thank you for CR.

**Comment by serious6** [comment]
@vbauer thanks for your contribution

**Pull request merged**